public/app/features/dashboard/components/DashboardSettings/GeneralSettings.tsx

```diff
@@ -115,6 +117,12 @@ export function GeneralSettingsUnconnected({
             />
           </Field>
 
+          {config.featureToggles.nestedFolderPicker && (
```

Comment:
Also, I cannot comment on line 106/108 but we should gate the other `FolderPicker` with `!config.featureToggles.nestedFolderPicker` , I don't suppose we want to show both 😉